workspaces/liferay-blank-workspace/configs/local/portal-ext.properties

```diff
@@ -0,0 +1,12 @@
+#
+# MySQL
+#
+#jdbc.default.driverClassName=com.mysql.cj.jdbc.Driver
+#jdbc.default.url=jdbc:mysql://localhost/lportal?useUnicode=true&characterEncoding=UTF-8&useFastDateParsing=false
+#jdbc.default.username=root
+#jdbc.default.password=
+
+feature.flag.LPS-164563=true
```

Comment:
Do we want this for all new workspaces done by blade init?  To enable all  of these CX related feature flags?

Reply:
We'll probably strip out the flags during blade init.